habitat_baselines/rl/ddppo/algo/ddppo_trainer.py

```diff
@@ -188,8 +201,10 @@ def train(self) -> None:
 
         observations = self.envs.reset()
         batch = batch_obs(observations, device=self.device)
+        batch = apply_obs_transforms_batch(batch, self.obs_transforms)
 
-        obs_space = self.envs.observation_spaces[0]
+        # obs_space = self.envs.observation_spaces[0]
+        obs_space = self.obs_space
```

Comment:
Maybe, Remove both lines, and use `self.obs_space` instead of `obs_space`

Reply:
This mutates the observation space in a very specific way though and the original code did not modify the observation space return from the envs so I am not sure that is the proper move.